qemu/tests/cfg/balloon_in_use.cfg

```diff
@@ -0,0 +1,93 @@
+- balloon_in_use:
+    type = driver_in_use
+    only Windows
+    extra_params += "-device virtio-balloon-pci,id=ballooning"
+    iterations = 5
+    ratio = 0.5
+    login_timeout = 800
+    start_vm = yes
+    kill_vm_on_error = yes
+    check_guest_bsod = yes
+    driver_name = "balloon"
+    run_bgstress = balloon_stress
+    bg_stress_run_flag = balloon_test
+    free_mem_cmd = cat /proc/meminfo |grep MemFree
```

Comment:
Hi, coli
Seems "free_mem_cmd" is not useful for windows guest. Could you pls check it？